packages/core/parcel-bundler/src/utils/md5.js

```diff
@@ -11,11 +11,13 @@ function md5(string, encoding = 'hex') {
 md5.file = function(filename) {
   return new Promise((resolve, reject) => {
     fs.createReadStream(filename)
+      .on('error', function(error) {
+        reject(error);
```

Comment:
I think you can write this like before by just passing `reject` directly as the callback instead of making a wrapper function